src/Store/MongoDB/Store.php

```diff
@@ -57,6 +58,26 @@ public function __construct(
     ) {
     }
 
+    public function initialize(array $options = []): void
+    {
+        $this->getCollection()->createSearchIndex(
+            definition: [
+                'fields' => [
+                    [
+                        'numDimensions' => 1536,
+                        'path' => $this->vectorFieldName,
+                        'similarity' => 'euclidean',
+                        'type' => 'vector',
+                    ],
+                ],
+            ],
+            options: [
+                'name' => $this->indexName,
+                'type' => 'vectorSearch',
+            ],
+        );
+    }
```

Comment:
@alcaeus, this looks like mentioned in:
https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-type/#replace-the-following-values-and-save-the-file.

But I receive:
![CleanShot 2024-09-26 at 23 04 03@2x](https://github.com/user-attachments/assets/204ce7ea-9a6a-463b-a398-1a49b99756cc)

Any clue?

Reply:
It works now with mongodb/mongodb 1.20

@alcaeus any remarks?